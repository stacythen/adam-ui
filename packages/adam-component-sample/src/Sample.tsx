import React from 'react';

interface ISample {
  /** properties description here */
  children: React.ReactNode;
}

/**
 * component description here
 */
const Sample = (props: ISample): React.ReactElement => {
  return <span>{props.children}</span>;
};

export default Sample;
