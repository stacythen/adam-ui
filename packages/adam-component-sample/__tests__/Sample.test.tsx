import React from 'react';
import { shallow } from 'enzyme';
import Sample from '../src/Sample';

describe('Sample', () => {
  it('should render properly', () => {
    const link = shallow(<Sample>Link to Google</Sample>);
    expect(link).toMatchSnapshot();
  });
});
