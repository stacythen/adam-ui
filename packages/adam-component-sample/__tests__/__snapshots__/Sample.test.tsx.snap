// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample should render properly 1`] = `
<span>
  Link to Google
</span>
`;
